<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-GHDB9E9" failures="7" tests="7" name="Test(failed)" time="47.8" errors="0" timestamp="27 Feb 2023 12:33:10 GMT">
  <testcase classname="testng.DataProviderExample" name="testSearchCountry" time="6.411">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testng.DataProviderExample.testSearchCountry(DataProviderExample.java:53)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testSearchCountry -->
  <testcase classname="testng.DataProviderExample" name="testSearchCountry" time="6.202">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testng.DataProviderExample.testSearchCountry(DataProviderExample.java:53)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testSearchCountry -->
  <testcase classname="testng.DataProviderExample" name="testSearchCountry" time="6.366">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testng.DataProviderExample.testSearchCountry(DataProviderExample.java:53)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testSearchCountry -->
  <testcase classname="testng.DataProviderExample" name="testSearchCountry" time="6.21">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testng.DataProviderExample.testSearchCountry(DataProviderExample.java:53)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testSearchCountry -->
  <testcase classname="testng.DataProviderExample" name="testSearchCountry" time="5.673">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testng.DataProviderExample.testSearchCountry(DataProviderExample.java:53)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testSearchCountry -->
  <testcase classname="testng.DataProviderExample" name="testSearchCountry" time="4.061">
    <failure type="org.openqa.selenium.StaleElementReferenceException" message="stale element reference: element is not attached to the page document
  (Session info: chrome=100.0.4896.60)
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: &amp;apos;3.141.5&amp;apos;, revision: &amp;apos;d54ebd709a&amp;apos;, time: &amp;apos;2018-11-06T11:42:16&amp;apos;
System info: host: &amp;apos;DESKTOP-GHDB9E9&amp;apos;, ip: &amp;apos;192.168.29.100&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;14.0.2&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 100.0.4896.60, chrome: {chromedriverVersion: 100.0.4896.60 (6a5d10861ce8..., userDataDir: C:\Users\Admin\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:59416}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: f6d3a7e4d9d3191acfc4e0d5c1a119d6">
      <![CDATA[org.openqa.selenium.StaleElementReferenceException: stale element reference: element is not attached to the page document
  (Session info: chrome=100.0.4896.60)
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '3.141.5', revision: 'd54ebd709a', time: '2018-11-06T11:42:16'
System info: host: 'DESKTOP-GHDB9E9', ip: '192.168.29.100', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '14.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 100.0.4896.60, chrome: {chromedriverVersion: 100.0.4896.60 (6a5d10861ce8..., userDataDir: C:\Users\Admin\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:59416}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: f6d3a7e4d9d3191acfc4e0d5c1a119d6
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:285)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:84)
at testng.DataProviderExample.testSearchCountry(DataProviderExample.java:48)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testSearchCountry -->
  <testcase classname="testng.DataProviderExample" name="testSearchCountry" time="5.915">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testng.DataProviderExample.testSearchCountry(DataProviderExample.java:53)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testSearchCountry -->
</testsuite> <!-- Test(failed) -->
